compoments/textInput.js

```diff
@@ -66,17 +66,17 @@ class TextInput extends Component {
   render(){
     return (
       <>
-        <h1 className="govuk-heading-l">{this.title}</h1>
+        <h1 className="govuk-heading-l" data-testid={`${this.name}-title`}>{this.title}</h1>
```

Comment:
I think that it makes sense to use `data-testid` in these cases as it's only used for testing.